tools/scraper/.gitignore

```diff
@@ -0,0 +1,5 @@
+cve-data/*
+data/*
+website-data/*
```

Comment:
At some point the data needs to work its way into the input directory and be included when the website is built. But that will come in future work.